preflight_check/check.py

```diff
@@ -1,17 +1,149 @@
 import re
-from . import utils
-from . import task
+import os
+import sys
+
 from crmsh import utils as crmshutils
 from crmsh import bootstrap as crmshboot
 from crmsh import completers
 
+from . import utils
+from . import task
+from . import config
+
+
+def fix(context):
+    """
+    Check configuration and fix the abnormal options
+    """
+    if context.check_conf:
+        can = check_sbd()
+        if can != "":
```

Comment:
I suggest here, we use a more clear name, instead of `can`, like `need_correct`, and the value of `need_correct` should be True/False, that is, we need also change the return type of `check_sbd`
@nick-wang  What do you think?

Reply:
Done with correct name.
The return string need to used later, so can't use `True/False` instead.